plugin.video.southpark_unofficial/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.southpark_unofficial" name="South Park" version="0.5.1" provider-name="Deroad">
+<addon id="plugin.video.southpark_unofficial" name="South Park" version="0.5.2" provider-name="Deroad">
```

Comment:
can you change the license to `GPL-2.0` please (spdx identifier)?

Reply:
why?

Reply:
I'll change the license to GPL2